### PR TITLE
#57 Alphavantagerepository creation

### PR DESCRIPTION
**Pull request by GurkomalSRao**
Test cases complete

### CODE REVIEW

**Review by jeffersoncasimir — APPROVED** [review]
Looks good. I added an invalid field to confirm that .isEmpty() is an appropriate method to validate the json object

File: src/main/java/nitezh/ministock/dataaccess/AlphaVantageRepository.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    private static final String API_KEY = "ZKD8M6L9CEQAK89H";
+    private String baseURL = "https://www.alphavantage.co/query?function=TIME_SERIES_";
+    private String cryptoBaseURL = "https://www.alphavantage.co/query?function=DIGITAL_CURRENCY_";
+    private final FxChangeRepository fxChangeRepository;
+
+    AlphaVantageRepository(FxChangeRepository fxChangeRepository) {
+        this.fxChangeRepository = fxChangeRepository;
+    }
+
+    /*public HashMap<String, StockQuote> getQuotes(Cache cache, String symbols) {
```
Comment (Nick9500) [inline src/main/java/nitezh/ministock/dataaccess/AlphaVantageRepository.java]: Remove the commented code please.

**Review by Nick9500 — CHANGES_REQUESTED** [review]
Good!

File: src/main/java/nitezh/ministock/dataaccess/AlphaVantageRepository.java (hunk 1 above)
Comment (GurkomalSRao) [reply src/main/java/nitezh/ministock/dataaccess/AlphaVantageRepository.java]: done

**Review by Nick9500 — APPROVED** [review]

**Pull request merged**